### PR TITLE
feat(unit-tests): unit tests for all components 

### PR DESCRIPTION
**Pull request by alebinson**

### CODE REVIEW

File: src/__mocks-data__/exportedPackages.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    sizeEst: 345,
+    tilesEst: 2000,
+    status: 'INPROGRESS',
+    link: 'https://packages/muku.gpkg',
+    date: new Date('2020-09-30T02:24:00'),
+    progress: 80,
+  },
+];
+
+export default MOCK_EXPORTED_PACKAGES;
```
Comment (asafMasa) [inline src/__mocks-data__/exportedPackages.ts:22]: end of line

File: src/__mocks__/confEnvShim.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      CHANNEL: 1002,
+      VERSION: 1,
+      REQUEST: 'REQUEST',
+      SERVICE_PROTOCOL: 'SERVICE_PROTOCOL', 
+      SERVICE_NAME: 'SERVICE_NAME',
+      ACTIVE_LAYER: 'ACTIVE_LAYER'
+      
+    };
+  })(void 0);
+}
```
Comment (asafMasa) [inline src/__mocks__/confEnvShim.js:15]: end of line

File: src/__mocks__/fileMock.js [hunk 3]
```diff
@@ -0,0 +1 @@
+module.exports = 'test-file-stub';
```
Comment (asafMasa) [inline src/__mocks__/fileMock.js:1]: end of line

File: src/__mocks__/styleMock.js [hunk 4]
```diff
@@ -0,0 +1 @@
+module.exports = {};
```
Comment (asafMasa) [inline src/__mocks__/styleMock.js:1]: end of line

File: src/common/helpers/estimated-tile-list.spec.ts [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+];
+
+
+describe('Polygon estimated tiles', () => {
+  it('return an array of estimated tiles',  () => {
+    const tilesArr = getTiles(polygonGeometry,minZoom, maxZoom);
+
+    expect(tilesArr).toEqual(RESULT);
+  });
+});
```
Comment (asafMasa) [inline src/common/helpers/estimated-tile-list.spec.ts:32]: end of line

File: src/exporter/components/export/export-dialog.spec.tsx [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+
+  //   wrapper.update();
+
+  //   await waitFor(() => {
+  //     const okButton = getButtonById(wrapper, 'general.ok-btn.text');
+  //     expect(okButton.prop('disabled')).toBe(true);
+  //   });
+  // });
+
+});
```
Comment (asafMasa) [inline src/exporter/components/export/export-dialog.spec.tsx:204]: end of line

File: src/exporter/models/exporterStore.ts [hunk 7]
```diff
@@ -10,11 +10,12 @@ import { Polygon } from '@turf/helpers';
...
 import { ResponseState } from '../../common/models/ResponseState';
 import { getLayerUrl } from '../../common/helpers/layer-url';
+// import MOCK_EXPORTED_PACKAGES from '../../__mocks-data__/exportedPackages';
 import { searchParams } from './search-params';
 import { IRootStore } from './rootStore';
-import { IGeoPackage } from './geoPackage';
+import { IExportTaskStatus } from './exportTaskStatus';
 
-export type GeoPackageResponse = IGeoPackage[];
+export type ExportTaskStatusResponse = IExportTaskStatus[];
```
Comment (asafMasa) [inline src/exporter/models/exporterStore.ts:18]: check eslint error Check warning on line 70 (result is not used)

**Review by asafMasa — CHANGES_REQUESTED** [review]
Great Job 💯 please mind my minor comments

File: src/__mocks-data__/exportedPackages.ts (hunk 1 above)
Comment (alebinson) [reply src/__mocks-data__/exportedPackages.ts:22]: Done

File: src/__mocks__/confEnvShim.js (hunk 2 above)
Comment (alebinson) [reply src/__mocks__/confEnvShim.js:15]: Done

File: src/__mocks__/fileMock.js (hunk 3 above)
Comment (alebinson) [reply src/__mocks__/fileMock.js:1]: Done

File: src/__mocks__/styleMock.js (hunk 4 above)
Comment (alebinson) [reply src/__mocks__/styleMock.js:1]: Done

File: src/common/helpers/estimated-tile-list.spec.ts (hunk 5 above)
Comment (alebinson) [reply src/common/helpers/estimated-tile-list.spec.ts:32]: Done

File: src/exporter/components/export/export-dialog.spec.tsx (hunk 6 above)
Comment (alebinson) [reply src/exporter/components/export/export-dialog.spec.tsx:204]: Done

File: src/exporter/models/exporterStore.ts (hunk 7 above)
Comment (alebinson) [reply src/exporter/models/exporterStore.ts:18]: Yes, I know. Leave it on purpose. Please neglect this warning

**Review by asafMasa — APPROVED** [review]

**Pull request merged**